Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -278,7 +278,7 @@ OptionSet<WebEventModifier> modifiersForNavigationAction(const NavigationAction&
 {
     OptionSet<WebEventModifier> modifiers;
     auto keyStateEventData = navigationAction.keyStateEventData();
-    if (keyStateEventData && keyStateEventData->isTrusted) {
+    if (keyStateEventData /*&& keyStateEventData->isTrusted*/) {
```

Comment:
I think this is about Safari UI behaviors, not really something that EWS would tell much about.

Reply:
Hopefully a test was added when we made this change.

Reply:
This code has changed too much and I had trouble finding the git history to find out what it's trying to fix.

Reply:
This is causing rdar://104342994 so it is important I know what this protected against to make sure I don't regress it.